### ISSUE TITLE
SDL_image config broken

### ISSUE THREAD

**Issue by davidgfnet** [pspdev/psplibraries#31]
Hey!
At compiling SDL_image I notice the following:

```
LIBPNG_CFLAGS='-I/usr/include/libpng16 '
LIBPNG_LIBS='-lpng16 '
```

And it compiles as:

```
/bin/sh ./libtool  --tag=CC   --mode=compile psp-gcc -DPACKAGE_NAME=\"\" -DPACKAGE_TARNAME=\"\" -DPACKAGE_VERSION=\"\" -DPACKAGE_STRING=\"\" -DPACKAGE_BUGREPORT=\"\" -DPACKAGE_URL=\"\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DLT_OBJDIR=\".libs/\" -DPACKAGE=\"SDL_image\" -DVERSION=\"1.2.12\" -DLOAD_JPG=1 -DLOAD_PNG=1 -DLOAD_BMP=1 -DLOAD_GIF=1 -DLOAD_LBM=1 -DLOAD_PCX=1 -DLOAD_PNM=1 -DLOAD_TGA=1 -DLOAD_XCF=1 -DLOAD_XPM=1 -DLOAD_XV=1 -I.     -I/usr/include/libpng16  -g -O2 -G0 -I"/usr/local/pspdev/psp/sdk/include" -I/usr/local/pspdev/psp/include/SDL -D_GNU_SOURCE=1 -Dmain=SDL_main -MT IMG_xv.lo -MD -MP -MF .deps/IMG_xv.Tpo -c -o IMG_xv.lo IMG_xv.c
```

(notice the /usr/include/libpng16). That is crearly wrong, it's picking up libpng files from my host system.
I noticed this cause I'm getting some crashes which print:

   libpng warning: Application built with libpng-1.6.10 but running with 1.5.7

That is, it's using the wrong headers at compile time, which makes apps crash :dancer: 
Could you give it a look? I'm no expert in auto tools otherwise I would submit a PR myself.

Thanks

**Comment by sharkwouter** [comment]
There are currently some issues with SDL_image not building, but I think this particular issue was resolved with #32

**Comment by dbeef** [comment]
Thanks @sharkwouter. I am additionally linking this issue to PR #43 - will close when done.
